# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/ncompress/install-symlink.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- ncompress-5.0/Makefile.def.vanilla	2023-01-22 22:33:00.726480098 +0100
+++ ncompress-5.0/Makefile.def	2023-01-22 22:33:26.847479782 +0100
@@ -44,7 +44,7 @@
 		mkdir -p $(DESTDIR)$(BINDIR) $(DESTDIR)$(MANDIR)
 		cp compress $(DESTDIR)$(BINDIR)/compress
 		rm -f $(DESTDIR)$(BINDIR)/uncompress
-		ln $(DESTDIR)$(BINDIR)/compress $(DESTDIR)$(BINDIR)/uncompress
+		ln -svf compress $(DESTDIR)$(BINDIR)/uncompress
 		cp compress.1 uncompress.1 $(DESTDIR)$(MANDIR)/.
 		chmod 0644 $(DESTDIR)$(MANDIR)/compress.1 $(DESTDIR)$(MANDIR)/uncompress.1
 
